### PR TITLE
FDS Source: fix error related to combination of periodic and Neumann …

### PR DESCRIPTION
**Pull request by SusanKilian**
I fixed an error related to the condensed system which only occurred when periodic and Neumann BC's are combined (in the absence of Dirichlet BCs).
Furthermore I temporarily deactivated an array only needed for the Algebraic Multigrid which isn't used currently.

### CODE REVIEW

File: Source/scrc.f90 [hunk 1]
```diff
@@ -387,7 +387,7 @@ MODULE SCRC
                       NSCARC_STACK_MAX               = 10            !> maximum number of consecutive solvers
 
 REAL(EB), PARAMETER:: NSCARC_THRESHOLD_CONVERGENCE   = 1.0E-15, &    !> threshold for convergence
```
Comment (rmcdermo) [inline Source/scrc.f90]: @SusanKilian All your (EB) parameters should be eight byte reals, for example, 1.0E-15_EB

Comment (rmcdermo) [reply Source/scrc.f90]: Also, what is your intention for 1.0E-16?  Is this machine precision?  If so, we use use TWO_EPSILON_EB, which is a global parameter.

**Comment by SusanKilian** [comment]
Oh, sorry, typing error with the missing _EB. I use TWO_EPSILON_EB from now on.

**Comment by rmcdermo** [comment]
OK, looks good.  Last nit pick (for next PR), you should not use >= when comparing with epsilon, just use >.

**Comment by SusanKilian** [comment]
Okay, makes sense ... I will take it into account.

**Pull request merged**